R/h_bracket.R

```diff
@@ -24,5 +24,5 @@ h_bracket <- function(x) {
 str_map <- function(i) {
     i <- gsub("^\\s*", "", i)
     i <- gsub("\n\\s*", "\n    ", i)
```

Comment:
```suggestion
    gsub("\n\\s*", "\n    ", i)
```
without `return` below would be standard (avoiding explicit `return`s)